.github/workflows/main.yml

```diff
@@ -0,0 +1,21 @@
+name: github-project-issue-to-sheets
+
+# Controls when the action will run. Triggers the workflow on push or pull request
+# events but only for the master branch
+on:
+  workflow_dispatch:
+  issues:
+    types: [opened, deleted, transferred, closed, reopened, assigned, unassigned, labeled, unlabeled]
+
+jobs:
+    github-project-issue-to-sheets:
+        runs-on: ubuntu-latest
+        name: github-project-issue-to-sheets
+        steps:
+        - name: github-project-issue-to-sheets
+          id: github-project-issue-to-sheets
+          uses: ViRGiL175/github-project-issue-to-sheets@dev
+          with:
+            google-api-service-account-credentials: ${{ secrets.GOOGLE_SERVICE_ACCOUNT_DATA }}
+            document-id: '1cuGeJlPIoKEo6su2vGx1wmkAi0l5o_7ZXCFH0dA5CAA'
```

Comment:
Perhaps we could move `document-id` to https://github.com/NeotokyoRebuild/neo/settings/variables/actions so it doesn't have to be hardcoded inside the action?

Reply:
Good point, thanks